cg/cli/upload/vogue.py

```diff
@@ -16,7 +16,7 @@
 
 LOG = logging.getLogger(__name__)
 
-VOGUE_VALID_BIOINFO = [str(Pipeline.MIP_DNA)]
```

Comment:
But should we not allow both production pipelines i.e. MIP-DNA, and Balsamic?

Reply:
yes, I'm testing for now. needed a working branch to test on Stage :-) I'm fixing it on vogue side in parallel as well.